docs/DeveloperGuide.md

```diff
@@ -36,25 +36,88 @@ would identify a, b and c from toDo.
 
 Step 3. To solve the quadratic equation, a, b and c are put into the formula `x = (-b±√(b^2-4ac))/2a`.
 
-### Store and Display Notes
-The Store and Display Notes functionality allows users to input notes 
-which will be stored in a list, and to display a list of all notes
-stored by users. This functionality is achieved through the `Store.` and `List.` commands.
+### Notes
+The Notes feature allows users to input their notes for storage purposes 
+and to keep track of their notes.
+
+This feature is implemented by storing the notes input from users into
+the data/notes.txt file, and users are able to keep track
+of their notes by reading the content from the data/notes.txt file/.
+
+#### Notes Text:
+The `notes.txt` is a text file that contains the list of Notes stored by users while using BadMaths.
+It is located in the `data` directory of the project.
+The purpose of this file is to allow users to keep a list of their notes stored 
+and to display their notes item stored as and when is needed for their educational purposes.
+
+#### Command class:
+Step 1. Define the `Command` class: The `Command` class takes in a command takes in a `command` and a `toDo` item and
+based on both of these inputs, the `Command` class then identifies the relevant functions
+to be executed.
+
+Step 2. The `executeCommand` method does the executing of the relevant functions depending on the `command` that
+was input by users. There are several functionalities provided in the `executeCommand` for users, including but
+not limited to the Storing and Deleting of Notes, Marking and Unmarking of notes items as completed or incomplete, and
+the searching for notes based through keywords or priorities of the notes. 
+
+#### Ui class:
+Step 1. Define the `Ui` class: The `Ui` class consists of several static methods which are used
+for the purposes of printing the user interface messages when users are performing the
+operations of BadMaths such as the notes and calculation operations and functionalities.
+
+The methods that are in the Ui class are as follows:
+`printIncorrectFormatEnteredelevant ()`: prints out a warning message to alert the 
```

Comment:
What is "Enteredelevant" in the method name? Suggest to check/change the method name.